Client/src/models/gui/ProfileGrid.java

```diff
@@ -15,8 +16,10 @@
     private static final String GENERAL_ICON_URL = "Client/resources/ui/icon_general.png";
     private static final String GOLD_ICON_URL = "Client/resources/ui/icon_gold.png";
     private static final String HISTORY_ICON_URL = "Client/resources/ui/icon_history.png";
-    private static final String USERNAME_TEXT = "Username:";
-    private static final String MATCH_HISTORY_TEXT = "Match Histories:";
+
+
+    private static final String USERNAME_TEXT = LanguageData.getInstance().getValue(new String[]{"LOGIN_MENU", "USERNAME"});
```

Comment:
Should probably use constants or an enum for these in all places so it's harder to mistype and have it compile.

Reply:
In principle I agree; I'd much rather have consts over strings,

However, its actually difficult to do that, this is a compromise I found.

Maybe it's just bad design on my part, but this was the simplest way I found to load in a language as a json file _(which, incidentally can have different types, which makes static typing difficult)._ 
  
I dunno, If you have the time check out the languageData class and see if you can think of a better design.

Reply:
#255 contains what I was talking about doing here. It's a PR to merge into this one.

Reply:
I do have an idea to make the whole thing statically typed using Optional (since they can't contain different ones, just subsets of english), but I'm leaving that for later.

Reply:
overall I'm happy with your changes.   Should we close this one and just merge yours?

Reply:
Mine is open against yours. I'll merge that, then you can merge this.